include/solvers/navier_stokes_scratch_data.h

```diff
@@ -483,6 +503,32 @@ class NavierStokesScratchData
       }
   }
 
+  template <typename VectorType>
+  void
+  reinit_pfg(const typename DoFHandler<dim>::active_cell_iterator &pfg_cell,
```

Comment:
again let's move on to a better name than pfg :)

Reply:
changed.